superduperdb/components/vector_index.py

```diff
@@ -170,6 +170,39 @@ def cleanup(self, db: Datalayer):
         db.fast_vector_searchers[self.identifier].drop()
         del db.fast_vector_searchers[self.identifier]
 
+    def on_db_event(self, db, events):
```

Comment:
I like this hook - we can use it for general `Triggers`.

Reply:
sure